starknet_py/tests/e2e/docs/code_examples/test_contract.py

```diff
@@ -66,7 +64,7 @@ async def test_declare_v2(account):
         account,
         compiled_contract=compiled_contract["sierra"],
         compiled_contract_casm=compiled_contract["casm"],
-        max_fee=int(1e15),
+        max_fee=int(1e18),
```

Comment:
What's the reason for this change? Btw, we can use `MAX_FEE` constant.

Reply:
The cost of declaration changed since this contract started being skipped, so it was failing due to `Max fee is smaller than the minimal transaction cost` error .
The content of this test code appears in the documentation, so I suppose that using `MAX_FEE` could be confusing

Reply:
Ok, I didn't see it's in docs, thanks for explanation.